packages/governance-app/src/Council/CouncilSummary.tsx

```diff
@@ -21,12 +22,12 @@ export function CouncilSummary () {
 
   useEffect(() => {
     const subscription = combineLatest([
-      (api.query.council.carryCount() as unknown as Observable<U32>),
-      (api.query.council.desiredSeats() as unknown as Observable<U32>),
-      (api.query.council.termDuration() as unknown as Observable<BlockNumber>),
-      (api.query.council.voteCount() as unknown as Observable<VoteIndex>),
-      (api.query.council.votingBond() as unknown as Observable<BalanceOf>),
-      (api.query.council.votingPeriod() as unknown as Observable<BlockNumber>)
+      (api.consts.elections.carryCount as unknown as Observable<u32>),
+      (api.consts.elections.desiredSeats as unknown as Observable<u32>),
+      (api.query.elections.termDuration() as unknown as Observable<BlockNumber>),
```

Comment:
`api.query.elections.termDuration<BlockNumber>()` also works. (Not 100% sure on `consts`, but could be the same)